### PR TITLE
Added basic MASON spread model

### PR DESCRIPTION
**Pull request by JeffGoldblum**
Added preliminary model of viral spread in MASON

### CODE REVIEW

**Comment by alexanian** [comment]
Would MASON make more sense as a subdirectory name than virus3? Or are we likely to work with other libraries?

Also could you add headers to the files? It is medium clear what they do but headers are friends.

File: spread/virus3/Cell.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            // The id is set in SimpleVirusDemo when a new agent is made
+            intID = Integer.parseInt( id.substring(4) ); // "Cell"
+            }
+        catch( Exception e )
+            {
+            throw new RuntimeException(e);
+            }
+        }
+    
+    public void step( final SimState state )
```
Comment (alexanian) [inline spread/virus3/Cell.java:37]: This is a nitpicky style thing, but can you make the empty braces concise? (e.g. http://google.github.io/styleguide/javaguide.html#s4.1.3-braces-empty-blocks)

Comment (JeffGoldblum) [inline spread/virus3/Cell.java:37]: Where do you see empty braces?

**Comment by JeffGoldblum** [comment]
Every time I want to run the model I have to type out the package name.
MASON_model is a lot to type out, so it's just "virus" for now.

As for the 3 at the end, it's my own form of version control because I'm
too lazy for PRs. Every time I manage to add in a few features without
completely breaking everything I leave that version alone and copy it to
start the next version. If anything breaks in this new version I have the
old one to look at and find the issue.

In conclusion, both parts of the name are because I'm lazy.

File: spread/virus3/Cell.java (hunk 1 above)
Comment (JeffGoldblum) [inline spread/virus3/Cell.java:37]: Oh, I was just looking at the code that was shown in the email...I'll get rid of the empty braces in future versions

**Comment by mattsmart** [comment]
Can we merge this? Do you have any pending commits? 

I'm also gonna change the directory structure so that it looks likes this:
models\viralspread\abm\mason\ or 
models\viralspread\abm\netlogo\

**Comment by alexanian** [comment]
I think we can merge esp. with the improved directory structure.

**Comment by mattsmart** [comment]
I'll make directory change in separate PR. Let us know if its good to go in @JeffGoldblum

**Comment by mattsmart** [comment]
Mergin'
